### PR TITLE
Feat/adds npm install and curl

### PR DESCRIPTION
**Pull request by johnnymatthews**
Adds NPM install commands to quickstart. Also adds a way for the reader to retrieve their file using Curl, if they so choose.

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

File: docs/README.md [hunk 1]
```diff
@@ -15,6 +15,14 @@ All you need to use Web3.Storage is an API token and some data. Web3.Storage hid
 
 Get up and running quickly by following these simple steps! We're going to sign up to Web3.Storage, get an API token, create a script to upload a file, and then view that file in the browser.
 
+:::tip Prerequisites
+All you need to complete this quickstart is:
+
+- Node version 14 or higher
+- NPM version 7 or higher
```
Comment (yusefnapora) [inline docs/README.md:22]:
```suggestion
- NPM version 6 or higher
```

**Review by yusefnapora — APPROVED** [review]
Very nice, just a nit that we only need npm 6+

File: docs/README.md (hunk 1 above)
Comment (johnnymatthews) [reply docs/README.md:22]: Nice. Thanks!

**Pull request merged**